pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -54,14 +54,15 @@ load_user_files <- function(input, pipeline_config, prev_out, input_dir = "/inpu
 read_10x_files <- function(config, input_dir) {
   counts_list <- list()
   annot_list <- list()
+  features_types_list <- list()
 
   samples <- config$samples
 
-
   for (sample in samples) {
     sample_dir <- file.path(input_dir, sample)
     sample_fpaths <- list.files(sample_dir)
     annot_fpath <- file.path(sample_dir, "features.tsv.gz")
+    gene_column <- 1
```

Comment:
ideally add constants either at the top of the file or a different one and use upper case (or any convention you are following) to make it more readable

Reply:
This is a variable though (gene_column)